apps/test-blinky/src/main.cpp

```diff
@@ -5,8 +5,8 @@
 
 #include "mbed.h"
 
-// Blinking rate in milliseconds
-#define BLINKING_RATE 500ms
+// Blinking rate in millisecond
+constexpr auto BLINKING_RATE = 500ms;
```

Comment:
Just to confirm - what type is passed into the sleep_for function that passes this variable?

Reply:
std::chrono::duration